src/server/req_delete.c

```diff
@@ -866,6 +878,10 @@ req_deletejob(struct batch_request *preq)
 			chk_array_doneness(parent);
 		}
 	}
+
+fn_exit:
```

Comment:
Two things about this label.  First, labels are scoped to functions.  This means you don't need to prefix it if you don't want to.  Second, I wouldn't call it exit.  You aren't exiting the program, you're basically returning.

I've seen 'done' used as a label name in other parts of the code.

In any case, the label name is fine if you want to keep it.

Reply:
I generally prefix the label with the scope to make it more, hence the 'fn' for function.  Your point about 'exit' is valid and I normally use fn_return and fn_error, the latter for handling exception like situations.  I'll rename the label once we figure out the correct solution.  The changes to req_delete.c are only a partial solution.  The main purpose of the PR was to submit failing tests and initiate a discussion on the right way to solve the issues.